client/state/stored-cards/actions.js

```diff
@@ -36,9 +36,12 @@ export const fetchStoredCards = () => dispatch => {
 	} );
 
 	return new Promise( ( resolve, reject ) => {
-		wp.undocumented().getStoredCards( ( error, data ) => {
-			error ? reject( error ) : resolve( data );
-		} );
+		wp.undocumented().getPaymentMethods(
+			{ type: 'all', status: 'stored', expired: 'exclude' },
+			( error, data ) => {
```

Comment:
If you exclude the callback, then this function will return a Promise, so you won't have to re-create one here. I know you're just following the existing code, but maybe whoever wrote that was not aware that `wpcom.req.get` returns a Promise without a callback, or maybe it didn't when that was originally written. So this might be a good time to simplify the code if you feel like it.